### PR TITLE
Persist dropdown and checkbox options for the monitoring page

### PR DESCRIPTION
**Pull request by jwforres**
Implements https://trello.com/c/vZbjaAwZ

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: app/scripts/controllers/monitoring.js [hunk 1]
```diff
@@ -73,10 +74,12 @@ angular.module('openshiftConsole')
...
       deployments: {},
       builds: {}
-    };    
+    };
+
+    var isNil = $filter('isNil');
 
     $scope.filters = {
-      hideOlderResources: true,
+      hideOlderResources: isNil($routeParams.hideOlderResources) ? true : $routeParams.hideOlderResources === 'true',
```
Comment (spadgett) [inline app/scripts/controllers/monitoring.js]:
```js
isNil($routeParams.hideOlderResources) || $routeParams.hideOlderResources === 'true'
```

File: app/scripts/controllers/monitoring.js [hunk 2]
```diff
@@ -32,7 +33,7 @@ angular.module('openshiftConsole')
 
     $scope.kindSelector = {
       selected: {
-        kind: "All"
+        kind: $routeParams.kind || "All"
```
Comment (spadgett) [inline app/scripts/controllers/monitoring.js]: Worth checking if `$routeParams.kind` is in options?

**Comment by jwforres** [comment]
[merge]

**Pull request merged**